crest/Assets/Crest/Crest/Scripts/LodData/OceanDepthCache.cs

```diff
@@ -77,11 +78,25 @@ public enum OceanDepthCacheRefreshMode
         bool _runValidationOnStart = true;
 #pragma warning restore 414
 
-        RenderTexture _cacheTexture;
-        public RenderTexture CacheTexture => _cacheTexture;
+        [Tooltip("Generate a signed distance field for shorelines"), SerializeField]
```

Comment:
The SDF options could go under a heading to keep it seperate from the other options, and adding something that makes it clear to users that this feature is not currently being used or that it is an upcoming feature?